.gitignore

```diff
@@ -1,5 +1,6 @@
 /rebuild.bat
 /doAllTheThings.bat
+/gradlew-*.bat
```

Comment:
I use a bunch of batch files prefixed with 'gradlew-' for things like setupDecompWorkspace, etc, so I've just ignored those, but... I can remove this from the PR if it's an issue.